net: l2_wifi: Add connect call with stored credentials. (#10434)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Wi-Fi
-----

* Added:

>   * Integration of Wi-Fi connectivity with Connection Manager connectivity API.

Reviewer: Not yet, sorry -- conn_mgr doesn't yet have documentation. I'm hoping to change that soon

>   * Integration of Wi-Fi connectivity with Connection Manager connectivity API.

Reviewer 2: @glarsennordic Is it not this? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/connectivity/bluetooth/api/connection_mgmt.html

>   * Integration of Wi-Fi connectivity with Connection Manager connectivity API.

Reviewer 2: If it is that link, then please use this:

```suggestion
  * Integration of Wi-Fi connectivity with the :ref:`Connection Management <zephyr:bluetooth-connection-management>` API.
```

Author: Marking this resolved since there is no documentation for conn_mgr yet

>   * Integration of Wi-Fi connectivity with Connection Manager connectivity API.

Reviewer: Correct, it is not that

`conn_mgr` lives here: https://github.com/zephyrproject-rtos/zephyr/tree/main/subsys/net/conn_mgr

>   * Integration of Wi-Fi connectivity with Connection Manager connectivity API.

Author: Marking this resolved since there is no documentation for conn_mgr yet.

Wi-Fi
-----

* Added:

  * Integration of Wi-Fi connectivity with Connection Manager connectivity API.

>   * The :kconfig:option:`CONFIG_NRF_WIFI_IF_AUTO_START` Kconfig option to enable an application to set/unset auto_starton an interface.

Reviewer 3: ```suggestion
  * The :kconfig:option:`CONFIG_NRF_WIFI_IF_AUTO_START` Kconfig option to enable an application to set/unset auto_start on an interface.
```

Author: Done.

>   * The :kconfig:option:`CONFIG_NRF_WIFI_IF_AUTO_START` Kconfig option to enable an application to set/unset auto_start on an interface.

Reviewer 2: ```suggestion
  * The :kconfig:option:`CONFIG_NRF_WIFI_IF_AUTO_START` Kconfig option to enable an application to set/unset AUTO_START on an interface.
```

Author: Done
